sentinelhub/config.py

```diff
@@ -90,17 +90,18 @@ def _parse_configuration(self, config):
 
             return config
 
-        @staticmethod
-        def get_config_file():
-            """ Checks if configuration file exists and returns its file path
+        def get_config_file(self):
+            """ Checks if configuration file exists and returns its file path.
+            If file doesn't exist it creates a default configurations file.
 
             :return: location of configuration file
             :rtype: str
             """
             config_file = os.path.join(os.path.dirname(__file__), 'config.json')
 
             if not os.path.isfile(config_file):
```

Comment:
No need to change, but using `isfile` will fail in (rather improbable) case of having an existing directory named 'config.json'...

Reply:
It does seem rather improbable, but thanks for pointing it out.

Reply:
I think that in such an improbable case it would be completely ok if it fails.